app/pages/patientdata/patientdata.js

```diff
@@ -192,7 +194,7 @@ export let PatientData = translate()(React.createClass({
     return this.renderChart();
   },
 
-  renderEmptyHeader: function(title = 'Preparing Chart Data') {
+  renderEmptyHeader: function(title = t('Preparing Chart Data')) {
```

Comment:
Using `t` in the function signature here solves the issue you were addressing, but is causing the automated `lint` test to fail.

I suggest setting the default title within the function body.  Something like:
```js
  renderEmptyHeader: function(title) {
    const { t } = this.props;
    const headerTitle = title || t('Preparing Chart Data');
    return (
      <Header
        chartType={'no-data'}
        inTransition={false}
        atMostRecent={false}
        title={headerTitle}
        ref="header"
      />
    );
  },
```